.linkcheckerrc

```diff
@@ -49,6 +50,8 @@ ignore =
   ^https://www\.mir\.wustl\.edu/employees/daniel-marcus/
   ^https://www\.mir\.wustl\.edu/employees/janine-bijsterbosch/
   ^https://www\.umassmed\.edu/news/news-archives/2022/02/david-kennedy-awarded-$6-million-repronim-brain-imaging-grant
+  ^.github/workflows/link-checker.yaml
+  ^https://github\.com/ReproNim/abcd-redcap2rs/*
```

Comment:
```suggestion
```
This link will be removed by https://github.com/ReproNim/repronim.org/pull/329

Reply:
that is good if it would, but would it happen before this PR ?  if not -- best to just do this and then remove later from this skips. otherwise we would drag having red CI

Reply:
Yes I'm fixing that PR now. ( now https://github.com/ReproNim/repronim.org/pull/338)

Reply:
In general, I prefer not to ignore broken links-- I'd rather CI be red if there are broken links to keep us honest.